api/document/tree.py

```diff
@@ -6,7 +6,7 @@
 
 from document.models import DocNode
 
-JsonDict = Dict[str, Any]
+PrimitiveDict = Dict[str, Any]
```

Comment:
Thanks for doing this; ideally we'd remove `Json` everywhere else (e.g. JSONAwareCursor), too.